### DISCUSSION TITLE
Any Support for HTTP REST API and Android?

### DISCUSSION THREAD

**Discussion by dipinraj** [sqlitecloud/discussions#2] [Feature request] [answered]
I couldn't find in documentation. Is there any support for HTTP Restful API?
All i could find was this ....(https://docs.sqlitecloud.io/docs/introduction/api).. It doesn't say anything much about the API

**Comment by marcobambini** [comment]
Hi @dipinraj next week we'll announce our new REST API functionalities.
You'll be able to write your own functions with Javascript or Python and/or use automatically generated OpenAPI for your databases and tables.

For Android development you can take a look at our Kotlin SDK:
https://github.com/sqlitecloud/kotlin

**Reply by bluemetaldragon** [reply, marked as answer]
Hi Marco, any idea by when will the REST APIs be available? Thanks

**Reply by marcobambini** [reply]
@bluemetaldragon @dipinraj REST APIs are already available in the new Development section of dashboard.sqlitecloud.io.
If you click Development, you can see a list of predefined REST APIs following the OpenAPI standard.
You can create your REST APIs in the Edge Functions section.
Documentation is under review and will be updated in a few days.
